lib/GPUArraysCore/src/GPUArraysCore.jl

```diff
@@ -209,19 +209,4 @@ macro allowscalar(ex)
     end
 end
 
-
-## other
-
-"""
-    backend(x)
-    backend(T::Type)
-
-Gets the GPUArrays back-end responsible for managing arrays of type `T`.
-"""
-backend(::Type) = error("This object is not a GPU array") # COV_EXCL_LINE
-backend(x) = backend(typeof(x))
-
-# WrappedArray from Adapt for Base wrappers.
-backend(::Type{WA}) where WA<:WrappedArray = backend(unwrap_type(WA))
```

Comment:
@vchuravy Does KA.jl already support recursing into wrapped arrays for `get_backend` queries?

Reply:
Uhm I don't think so, but we can add that.

Reply:
I'm not entirely sure we want to; it would pull the whole Union mess of Adapt's `WrappedArray` into KA.jl. On the other hand, there's not much of an alternative right now if we want the ability to launch kernels on wrapped arrays...